packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -33,9 +33,41 @@
   overflow: hidden;
 }
 
+.mdc-chip--with-leading-icon {
+  padding-top: $mdc-chip-with-icon-padding;
+  padding-bottom: $mdc-chip-with-icon-padding;
+  padding-left: $mdc-chip-with-icon-padding;
+}
+
+.mdc-chip--with-trailing-icon {
+  padding-right: $mdc-chip-with-icon-padding;
+}
+
 .mdc-chip__text {
   display: inline-block;
   vertical-align: middle;
   // Set line-height to be <18px to force the content height of mdc-chip to be 18px.
   line-height: 17px;
 }
+
+.mdc-chip__icon {
+  color: $mdc-chip-icon-color;
+  vertical-align: middle;
+  margin: 0 4px;
+  border-radius: 16px;
```

Comment:
`border-radius: 50%` should do the trick, and it avoids hard-coding a specific pixel value 😀